packages/next/build/webpack/loaders/next-serverless-loader.ts

```diff
@@ -215,13 +217,14 @@ const nextServerlessLoader: loader.Loader = function() {
       }
       let _nextData = false
 
-      if (req.url.match(/_next\\/data/)) {
+      const parsedUrl = handleRewrites(parse(req.url, true))
```

Comment:
Nice trick, looks like we were blowing away the query before. Want to do this in a separate PR so we can isolate the test (that it's still removed for SSG data/page routes)?

Reply:
Opened PR here: https://github.com/zeit/next.js/pull/10261